docs/api/api_docs.yaml

```
@@ -976,27 +976,111 @@ paths:
     get:
       responses:
         '200':
-          description: OK
+          description: Found attachment successfully.
+          headers:
+            Etag:
+              schema:
+                type: string
+              description: 'The attachment digest. Does not get set when request `meta=true`. '
+        '206':
+          description: Partial attachment content returned
+        '404':
+          $ref: '#/components/responses/Not-found'
+        '416':
+          description: Requested range exceeds content length
       tags:
         - Admin
         - Public
+      summary: Get an attachment from a document
+      description: |-
+        This request retrieves a file attachment associated with the document. 
+
+        The raw data of the associated attachment is returned (just as if you were accessing a static file). The `Content-Type` response header is the same content type set when the document attachment was added to the database. The `Content-Disposition` response header will be set instead if the content type is vulnerable to a phishing attack, causing the browser to download the file instead of display it (unless overriden by by database config option `serve_insecure_attachment_types`).
+
+        If the `meta` query parameter is set then the response will be in JSON with the additional metadata tags.
+      parameters:
+        - $ref: '#/components/parameters/rev'
+        - schema:
+            type: boolean
+            default: 'false'
+          in: query
+          name: meta
+          description: Include the metadata of the attachment in the response body. Will cause `Content-Type=application/json` in the response header.
     put:
       responses:
-        '200':
-          description: OK
+        '201':
+          description: Attachment added to new or existing document successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/New-revision'
+          headers:
+            Etag:
+              schema:
+                type: string
+              description: The ID of the new revision.
+        '404':
+          $ref: '#/components/responses/Not-found'
+        '409':
+          $ref: '#/components/responses/Conflict'
       tags:
         - Admin
         - Public
+      summary: Create or update an attachment on a document
+      description: |-
+        This request adds or updates an attachment associated with the document. If the document does not exist, it will be created and the attachment will be added to it.
+
+        The maximum content size of an attachment is 20MB. The `Content-Type` header of the request should be what the content type of the attachment is otherwise this will default to `application/octet-stream`.
```

```suggestion
        The maximum content size of an attachment is 20MB. The `Content-Type` header of the request specifies the content type of the attachment.
```

Reword and default values are described already in the spec. I don't think there's a need to repeat them in a description.